blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/PipelineApiTest.java

```diff
@@ -797,13 +799,13 @@ public void parameterizedFreestyleTestWithoutDefaultParam() throws Exception {
                 ), 400);
     }
 
+    @Ignore
```

Comment:
@michaelneale @imeredith I spent about an hour trying to find whats going on. I kind of know whats going on, the failure is: 

```
1) No implementation for org.eclipse.aether.RepositorySystem was bound.
  while locating org.apache.maven.artifact.resolver.DefaultArtifactResolver
  at ClassRealm[maven, parent: ClassRealm[maven-parent, parent: null]] (via modules: org.eclipse.sisu.wire.WireModule -> org.eclipse.sisu.plexus.PlexusBindingModule)
  at ClassRealm[maven, parent: ClassRealm[maven-parent, parent: null]] (via modules: org.eclipse.sisu.wire.WireModule -> org.eclipse.sisu.plexus.PlexusBindingModule)
  while locating org.apache.maven.artifact.resolver.ArtifactResolver
  while locating org.apache.maven.repository.legacy.LegacyRepositorySystem
  at ClassRealm[maven, parent: ClassRealm[maven-parent, parent: null]] (via modules: org.eclipse.sisu.wire.WireModule -> org.eclipse.sisu.plexus.PlexusBindingModule)
  at ClassRealm[maven, parent: ClassRealm[maven-parent, parent: null]] (via modules: org.eclipse.sisu.wire.WireModule -> org.eclipse.sisu.plexus.PlexusBindingModule)
  while locating org.apache.maven.repository.RepositorySystem
  while locating org.apache.maven.execution.DefaultMavenExecutionRequestPopulator
  at ClassRealm[maven, parent: ClassRealm[maven-parent, parent: null]] (via modules: org.eclipse.sisu.wire.WireModule -> org.eclipse.sisu.plexus.PlexusBindingModule)
  at ClassRealm[maven, parent: ClassRealm[maven-parent, parent: null]] (via modules: org.eclipse.sisu.wire.WireModule -> org.eclipse.sisu.plexus.PlexusBindingModule)
  while locating org.apache.maven.execution.MavenExecutionRequestPopulator

1 error
      role: org.apache.maven.execution.MavenExecutionRequestPopulator
  roleHint: 
	at hudson.maven.MavenEmbedder.<init>(MavenEmbedder.java:130)
	at hudson.maven.MavenEmbedder.<init>(MavenEmbedder.java:109)
	at hudson.maven.MavenEmbedder.<init>(MavenEmbedder.java:136)
	at hudson.maven.MavenUtil.createEmbedder(MavenUtil.java:212)
	at hudson.maven.MavenModuleSetBuild$PomParser.invoke(MavenModuleSetBuild.java:1320)
	at hudson.maven.MavenModuleSetBuild$PomParser.invoke(MavenModuleSetBuild.java:1123)
	at hudson.FilePath.act(FilePath.java:996)
	at hudson.FilePath.act(FilePath.java:974)
	at hudson.maven.MavenModuleSetBuild$MavenModuleSetBuildExecution.parsePoms(MavenModuleSetBuild.java:985)
	at hudson.maven.MavenModuleSetBuild$MavenModuleSetBuildExecution.doRun(MavenModuleSetBuild.java:690)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:534)
	at hudson.model.Run.execute(Run.java:1728)
	at hudson.maven.MavenModuleSetBuild.run(MavenModuleSetBuild.java:542)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:405)
Caused by: org.codehaus.plexus.component.repository.exception.ComponentLookupException: com.google.inject.ProvisionException: Unable to provision, see the following errors:
```

There is circular dependency injection, `DefaultMavenExecutionRequestPopulator` -> `RepositorySystem`-> `LegacyRepositorySystem`->`ArtifactResolver`->`DefaultArtifactResolver`->`RepositorySystem` --- at this stage it fails with 

`No implementation for org.eclipse.aether.RepositorySystem was bound.`

Not sure its worth our time to make `MavenModuleSet` project work with our test system - there are lots of variable to consider. This test is not really adding much value either. I think we should delete this test.